src/main/java/hw_8/Task_1/Application.java

```diff
@@ -0,0 +1,74 @@
+package hw_8.Task_1;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Map<String, Person> personMap = new HashMap<>();
+        Person onePerson = new Person(25, "Oleg", "Vahnenko");
+        Person twoPerson = new Person(35, "Nana", "Petroff");
+        personMap.put("AAAA", onePerson);
+        personMap.put("MMM", twoPerson);
+        System.out.println("Enter command");
+        String addInformation = scanner.nextLine();
```

Comment:
variable name is related to "addCommand" but in fact there can be "getCommand" or "Exit". So rename this variable name to "command"